### ISSUE TITLE
sirix-keycloak-1  | User with username 'admin' already added to '/opt/jboss/keycloak/standalone/configuration/keycloak-add-user.json'

### ISSUE THREAD

**Issue by AlvinKuruvilla** [sirixdb/sirix#581]
**Describe the bug**
When running sirix with ```docker compose up``` or ```docker compose up keycloak``` it initially adds an admin user to the JSON. However, if afterwards you stop the container and rerun it, keycloak will continuously try to add the admin user again and keep failing, giving output like this:
<img width="833" alt="image" src="https://user-images.githubusercontent.com/46870885/220377091-9b3e2c39-f088-4913-b737-2870f75b9130.png">

**To Reproduce**
1. Run ```docker compose up``` or ```docker compose up keycloak```
2. After everything is setup stop the container and rerun the command again 

**Expected behavior**
I would suspect that it should not fail if the admin user already exists

**Additional context**
I know in the error it gives a path to /opt where a JSON file is; I tried looking for it there but all I could find was an empty sirix-data folder 
This issue also makes it impossible to visit the keycloak web UI

**Comment by JohannesLichtenberger** [comment]
Can you check what’s wrong in the proposed fix in https://github.com/sirixdb/sirix/issues/541 and the PR?

**Comment by AlvinKuruvilla** [comment]
I'll take a look thanks

**Comment by AlvinKuruvilla** [comment]
Thanks for the suggestion @JohannesLichtenberger the PR fixed the issue I was having, I think we should consider merging that in.

**Comment by JohannesLichtenberger** [comment]
The `sirix-rest-api` tests failed, as I think the user will not be created anymore, IIRC. Can you confirm that or make a new PR and we can have a look?

**Comment by AlvinKuruvilla** [comment]
I went into ```bundles/sirix-rest-api``` and then ran ```gradle build -x test``` like the README suggests and I get the following error:
<img width="867" alt="image" src="https://user-images.githubusercontent.com/46870885/220793040-7cd03fea-d79f-465b-910f-011704aa53e7.png">

**Comment by redwanulsourav** [comment]
For future references, I also followed same issue while working on the project on Windows using Docker. According to this [answer](https://stackoverflow.com/a/60067870) on stackoverflow, it happens if keycloak was interrupted during boot. But for my case my keycloak wasn't interrupted still I got the same error.

The fix mentioned in the [answer](https://stackoverflow.com/a/60067870) worked for me.

I also noticed that this didn't happen when I switched to a Linux machine. (Though it happen only once, then I stopped the container and restarted it and it was fixed)
